osu.Desktop.VisualTests/VisualTestGame.cs

```diff
@@ -14,7 +14,7 @@ protected override void LoadComplete()
         {
             base.LoadComplete();
 
-            new BackgroundScreenDefault { Depth = 10 }.LoadAsync(this, AddInternal);
+            LoadComponentAsync(new BackgroundScreenDefault { Depth = 10 }, AddInternal);

```

Comment:
I still disagree with this style of loading async. I think `LoadComponentAsync` should _always_ do `AddInternal` at the end; otherwise what you are loading is not a "component".
If someone wants a custom callback, they can still call `LoadAsync` manually (and also pass in a clock, because the container's clock doesn't make much sense if you don't `AddInternal`).